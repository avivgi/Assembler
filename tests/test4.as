.define xxx = 6
LABELK: .data 23,24,xxx,26,27,28
ANOTHER: .data 33,35,37
AVIV: .data 4
Moshe: .string "abcdefgh"
DAVID: .string "gttrfdert"
.extern HELLO
.entry BYE
mov a,b,c
BIL a,b,c
