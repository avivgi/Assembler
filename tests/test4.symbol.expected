Symbol name: xxx       	Value: 6	type: 1: MDEFINE
Symbol name: LABELK    	Value: 0	type: 3: DATA
Symbol name: ANOTHER   	Value: 6	type: 3: DATA
Symbol name: AVIV      	Value: 9	type: 3: DATA
Symbol name: Moshe     	Value: 10	type: 3: DATA
Symbol name: DAVID     	Value: 19	type: 3: DATA
Symbol name: HELLO     	Value: 0	type: 5: EXTERN
